### PR TITLE
feat: update Android Zendesk module to allow sending of tags

### PR DESCRIPTION
**Pull request by zoofan**

### CODE REVIEW

File: android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java [hunk 1]
```diff
@@ -29,10 +37,11 @@ public String getName() {
 
     @ReactMethod
     public void setVisitorInfo(ReadableMap options) {
-        VisitorInfo.Builder builder = new VisitorInfo.Builder();
+        VisitorInfo.Builder builder = new 
+        VisitorInfo.Builder();
 
         if (options.hasKey("name")) {
-            builder.name(options.getString("name"));
+            builder.name("name");
```
Comment (paustria) [inline android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java]: Why do you update this?

File: android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java [hunk 2]
```diff
@@ -54,9 +65,37 @@ public void init(String key) {
     @ReactMethod
     public void startChat(ReadableMap options) {
         setVisitorInfo(options);
-        Activity activity = getCurrentActivity();
-        if (activity != null) {
-            activity.startActivity(new Intent(mReactContext, ZopimChatActivity.class));
+        String[] tags = getTags(options);
+
+        PreChatForm preChatForm = new PreChatForm.Builder()
```
Comment (paustria) [inline android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java]: I don't think we need prechat form.

**Review by paustria — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java (hunk 1 above)
Comment (zoofan) [reply android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java]: Fixed.

File: android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java (hunk 2 above)
Comment (zoofan) [reply android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java]: You're right. Removed.

**Comment by paustria** [comment]
@zoofan Could you please resolve conflict?

**Review by binhxn — APPROVED** [review]

**Review by paustria — APPROVED** [review]

**Pull request merged**